docs/ref/checks.txt

```diff
@@ -315,6 +316,7 @@ Related fields
   the table name of ``<model>``/``<model>.<field name>``.
 * **fields.W345**: ``related_name`` has no effect on ``ManyToManyField`` with a
   symmetrical relationship, e.g. to "self".
+* **fields.W346**: ``ManyToManyField`` does not support ``db_comment``.
```

Comment:
Documentation doesn't match the raised warning.

Reply:
Please check again if you missed 
![스크린샷 2022-11-04 오후 7 54 02](https://user-images.githubusercontent.com/24240623/199956118-e3c68411-5419-470c-8ed0-f788917381e4.png)

Reply:
but the message is different